src/qt/clientmodel.cpp

```diff
@@ -46,6 +56,10 @@ ClientModel::ClientModel(interfaces::Node& node, OptionsModel *_optionsModel, QO
 ClientModel::~ClientModel()
 {
     unsubscribeFromCoreSignals();
+
+    delete pollTimer;
```

Comment:
why is the timer deleted before the thread stops and not the other way around?

Reply:
What is the issue?

Reply:
I don't know if there is an issue or not, I am just wondering if there is any specific reason for this ordering. Mostly because I saw [this comment](https://stackoverflow.com/questions/10492480/starting-qtimer-in-a-qthread#comment73988017_10493492).

Reply:
Thanks for pointing it, there's indeed an issue here - must be deleteLater of move after joining the other thread.

Reply:
Fixed.